phpstan.neon.dist

```diff
@@ -1,5 +1,5 @@
 parameters:
-  phpVersion: 70400
+  phpVersion: 80100
```

Comment:
The minimum PHP version in the `composer.json` is still 8.0 so I would like to keep it consistent here.

```suggestion
  phpVersion: 80000
```